certbot/storage.py

```diff
@@ -810,6 +810,19 @@ def new_lineage(cls, lineagename, cert, privkey, chain, cli_config):
             logger.debug("Writing full chain to %s.", target["fullchain"])
             f.write(cert + chain)
 
+        # Write a README file to the live directory
+        readme_path = os.path.join(live_dir, "README")
+        with open(readme_path, "w") as f:
+            logger.debug("Writing README to %s.", readme_path)
+            f.write("The files in this directory are symlinks to the latest "
+                    "versions of your keys and certificates.\n\n"
+                    "`privkey.pem` is the private key for your certificate.\n"
+                    "`fullchain.pem` contains all the information your certificate needs, and "
+                    "is likely the certificate file you will need.\n"
+                    "`chain.pem` is used for OCSP stapling in Nginx >=1.3.7.\n\n"
+                    "For more information, see the Certbot User Guide "
+                    "at https://certbot.eff.org/docs/using.html.\n")
```

Comment:
I'm good with breaking it out, but my goal here was to keep it as short as possible and point people to the online docs, because I don't want to have to be keeping this up to date in sync with the online docs.